content/blog/16-17/linux.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Linux: hoe doe?"
+created_at: 02-02-2017
+description: Uitleg en tips voor iedereen die linux wil installeren.
+author: Rien
+---
+
+_Tijdens het begin van ieder schooljaar komt er een verse lading studentjes Informatica toe die in de lessen Computergebruik kennis maken met de wondere wereld van GNU/Linux. Velen onder hen worden geprikkeld door de onontgonnen mysteries van dit besturingssysteem en groeien een verlangen om de kunst van het Unix-tovenaarschap zich eigen te maken._
+
+Deze blogpost dient als een leidraad voor de dappersten onder hen: zij die het pad naar de verlossing willen bewandelen en een eigen Linux-installatie willen tot leven wekken.
+
+# Enkele tips voor je begint
+---
+
+## Probeer eerst in een virtuele machine
+Als je nog geen ervaring hebt met het installeren van Linux start je best door eens te oefenen op een VM (virtuele machine). Zo komt niet alles in één keer op je af. Zo kun je eerst het installatieproces onder de knie krijgen zonder meteen rekening te hoeven houden met eventuele problemen, dual boot opzetten en andere obstakels.
+
+## Zit je vast? RTFM!
+Read The Fucking Manual (of forum, of wiki). Kom je op een rare error terecht, weet je niet wat de volgende stap is of weet je niet waar te beginnen? Raak niet direct in paniek, ieder struikelblok is een kans om te leren. Foutmeldingen geven vaak een aanwijzing waar het probleem zou kunnen liggen, lees die dus goed. Indien dat geen verlichting brengt: zoek je specifiek probleem op. De kans is klein dat jij de eerste bent met hetzelfde probleem.
+
+Enkele goede bronnen van informatie:
+- Google en [DuckDuckGo](https://duckduckgo.org) zijn uw vriend!
+- De [wiki](https://wiki.archlinux.org/) of het [forum](https://bbs.archlinux.org/) van Arch Linux. Veel algemene concepten en problemen worden hier uitgelegd die toepasbaar zijn op veel andere distro's.
+- Een online [manpage](http://man.he.net/).
+
+## Kies een distro
+Er zijn veel verschillende Linux-distributies, elk met een verschillende _look & feel_, moeilijkheidsgraad en ideologie. Voor beginners kan ik *Linux Mint* of *Fedora* aanraden. Wie al wat meer ervaring heeft en niet bang is van een terminal kan eens een kijkje nemen naar *Arch Linux*. [Hier](https://linuxjourney.com/lesson/linux-history#) vind je uitleg over de courantste distro's en op [distrowatch](https://distrowatch.com/) staan de meeste distributies met een korte uitleg.
+
+
+# 1. Voorbereiding
+---
+
+## Fix windows
+
+Als je van plan bent een dual-boot te doen (Linux en Windows op één machine) moet je rekening houden met het volgende:
```

Comment:
Ik zal het wat duidelijker maken dat dit document meer gaat over het effectief installeren. Eerst proberen in een VM is eerder een suggestie.